grammar/SubstraitLexer.g4

```diff
@@ -0,0 +1,157 @@
+lexer grammar SubstraitLexer;
+
+// Whitespace and comment handling
+LineComment   : '//' ~[\r\n]* -> channel(HIDDEN) ;
+BlockComment  : ( '/*' ( ~'*' | '*'+ ~[*/] ) '*'* '*/' ) -> channel(HIDDEN) ;
+Whitespace    : [ \t\r]+ -> channel(HIDDEN) ;
+
+// Substrait is case-insensitive, ANTLR is not. So, in order to define our
+// keywords in a somewhat readable way, we have to define these shortcuts.
+
+fragment A : [aA];
```

Comment:
@EpsilonPrime suggested swapping this with `caseInsensitive = true;` option over in substrait-io/substrait-python#63

Reply:
fixed it.